### ISSUE TITLE
[STUDIO][1.1.0-beta] Ajout d'une image

### ISSUE THREAD

**Issue by WITNicolas** [witsa/synapps#439]
Quand on rajoute une image, la liste déroulante se ferme.
Si on rajoute 1 seule image ce n'est effectivement pas gênant, si on veut en rajouter plusieurs c'est plus contraignant de devoir ré-ouvrir à chaque fois.
Cette 2eme hypothèse sera celle la plus souvent utilisée, ça sera très rare de ne rajouter qu'1 seule image dans un projet.

![image](https://user-images.githubusercontent.com/33632664/157048046-c820a670-87d8-47c2-805f-909ff3d5d661.png)

**Comment by manufitoussiwit** [comment]
Il y a le bouton Ajouter toutes les images qui peut peut-etre aider dans cette situation. Mais dans ce cas, il ne faut placer dans le dossier que celles qu'on veut vraiment.

**Comment by WITNicolas** [comment]
Allez je te l'accorde :) On clôture...

**Comment by manufitoussiwit** [comment]
Non, je vais quand même voir si je peux conserver la popup ouverte ;-)

> Allez je te l'accorde :) On clôture...

**Comment by Raphwit** [comment]
Corrigé dans la version [https://witsa.github.io/synapps/synapps-studio-releases/notes/1.3.0#fermeture-du-menu-dajout-image-ou-de-fichier-dinclusion](1.3.0)
